pgmax/fg/graph.py

```diff
@@ -235,21 +282,26 @@ def factor_groups(self) -> Tuple[groups.FactorGroup, ...]:
 
     @cached_property
     def fg_state(self) -> FactorGraphState:
-        """Current factor graph state given the added factors."""
+        """Current factor graph state given the added factors.
+        Also computes the factor_to_msgs_starts and the log_potentials."""
+        # Preliminary computations
+        log_potentials = self.log_potentials
+        wiring = self.wiring
```

Comment:
This does not seem to be changing anything. Better to get rid of it to avoid unnecessary diffs.

Reply:
This is necessary because `wiring` is computing `_factor_to_msgs_starts`